BUILDING.md

```diff
@@ -82,6 +82,10 @@ This project is built on [GitHub Actions](https://github.com/arduino/arduino-ide
     git push origin 1.2.3
    ```
 
+## Notes for Windows contributors
```

Comment:
I suggest moving this section inside ["**Build from source**"](https://github.com/arduino/arduino-ide/blob/5e1d0d00c8f30c0299b68b27c6318aa07e858848/BUILDING.md#build-from-source), above the ["**CI**"](https://github.com/arduino/arduino-ide/blob/5e1d0d00c8f30c0299b68b27c6318aa07e858848/BUILDING.md#ci) section. This information is about a dependency for building from source, so that is the most appropriate place for it.